christfellowshipchurch/web-app-v2
# Changed message icon to chat icon

### About
Updated mobile web SMS share button with chat icon instead of phone.

### Test Instructions
Open an article on mobile web and try to share it, the Message icon should be a chat icon, as seen on the screenshot below.

### Screenshots
<img width="203" alt="Screen Shot 2021-08-31 at 1 24 27 PM" src="https://user-images.githubusercontent.com/46769629/131548164-25735c6a-14f0-433e-99aa-073646b96a0e.png">

### Closes Tickets
[[CFDP-1677]](https://christfellowshipchurch.atlassian.net/browse/CFDP-1677)

### Related Tickets
N/A
